packages/kerosene-test/src/stubs/createStubStyles.ts

```diff
@@ -0,0 +1,45 @@
+import { kebabCase } from "lodash";
+
+/**
+ * Returns an object mapping local classNames to faked global equivalents along with any exported values.
+ * @param classNames List of local classNames from styles file
+ * @param values Map of exported values (values MUST be strings)
+ */
+export default function createStubStyles<C extends string, V extends {}>(
+  classNames: C[],
+  values = {} as V,
+): V & { [key in C]: string } {
+  Object.entries(values).forEach(([property, value]) => {
+    const type = typeof value;
+    if (type !== "string") {
+      throw new TypeError(
+        `The property: '${property}' with value '${value}' in createStubStyles(, values) must be a string, but was ${type}`,
+      );
+    }
+  });
+
+  const obj = classNames.reduce<V & { [key in C]: string }>(
+    (acc, className) =>
+      ({
+        ...(acc as object),
+        [className]: `${kebabCase(className)}-local-class ${kebabCase(
+          className,
+        )}-webpack-composed-class`,
+      } as V & { [key in C]: string }),
+    values as any,
+  );
+
+  // Use a Proxy to throw an error when trying to access styles which haven't been stubbed
+  return new Proxy(obj, {
```

Comment:
is proxy available on the Node versions we specify in `package.json`?

Reply:
Yes. `Proxy` was added in Node 6.